src/Utility.ts

```diff
@@ -87,11 +87,13 @@ export namespace Utility {
         if (config) {
             // tslint:disable-next-line:no-backbone-get-set-outside-model
             const workspace: string = config.get<string>('workspace');
-            try {
-                await fse.ensureDir(workspace);
-                return workspace;
-            } catch (err) {
-                return path.join(defaultStoragePath, 'tomcat');
+            if (!workspace.startsWith('<<')) {
+                try {
+                    await fse.ensureDir(workspace);
+                    return workspace;
+                } catch (err) {
+                    return path.join(defaultStoragePath, 'tomcat');
```

Comment:
Just swallow the error here since there is already `return path.join(defaultStoragePath, 'tomcat');` below

Reply:
yes, nice catch

Reply:
And, it seems there is not that necessary to try catch here...